### ISSUE TITLE
Text invisible in modal overlays

### ISSUE THREAD

**Issue by abrthel** [arcticicestudio/nord-atom-ui#53]
When trying to create a new file or folder the text in the window doesn't show up.  However when I focus on another window other than atom the text in the prompt is visible.

![capture](https://cloud.githubusercontent.com/assets/1217207/22273395/bb5ca3c6-e25d-11e6-88b8-3d0ccd74bb29.PNG)

```
Environment
Atom: v1.13.1
nord-atom-ui: v0.10.0
nord-atom-syntax: v0.6.0
```

**Comment by arcticicestudio** [comment]
Thanks for your report 👍 
Seems like this is related to the Atom theme package API changes regarding the shadow DOM removal.
I had to adjust the colors of `atom-text-editor[mini]` to fix the find-and-replace text box, but it seems like there are edge cases the color must be wrapped into the specific selector class like for this bug it must be wrapped into the `modal` class.
I'll fix this as soon as possible and release a patch version.

**Comment by arcticicestudio** [comment]
🚢 Fixed and in apm package release version 🏷 [0.10.1](https://github.com/arcticicestudio/nord-atom-ui/releases/tag/v0.10.1).

<p align="center"><strong>Before</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/22285966/ed31d25a-e2ed-11e6-9355-c53db4fec0d5.png"/><br><strong>After</strong><br><img src="https://cloud.githubusercontent.com/assets/7836623/22285980/f4cfede4-e2ed-11e6-9906-fa732ca787de.png"/></p>
